### ISSUE TITLE
Allow clickable elements to be activated

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/test_nvdaTracImport#115]
**Reported by jteh on 7 Mar 2008 02:04 UTC**
It is possible for any HTML element to have an onClick attribute which specifies code to run when the mouse is clicked. NVDA should allow these elements to be activated when enter is pressed.

**Comment by michaelDCurran** [comment]
**Modified by jteh on 28 Apr 2008 07:47 UTC**

**Comment by michaelDCurran** [comment]
**Comment by jteh on 19 May 2008 23:40 UTC**
 * Reporting and activation of clickable elements are separate issues. #91 has now been opened for reporting of clickables. I'm modifying this ticket accordingly.
 * MozillaBug:420863 has been fixed for a while now, so the click action is now exposed on elements themselves rather than their text leaf nodes.
 * Activation of clickable elements has always worked to some extent. However, due to MozillaBug:432603, it fails for many element types. A work around was implemented in NVDA in r2050. This ticket can therefore be closed.
